app/src/main/java/org/p2p/wallet/auth/ui/phone/countrypicker/CountryCodePickerPresenter.kt

```diff
@@ -16,6 +16,7 @@ class CountryCodePickerPresenter(
     BasePresenter<CountryCodePickerContract.View>(),
     CountryCodePickerContract.Presenter {
 
+    private var selectedCountryCode: CountryCode? = null
```

Comment:
don't we have defaultCountryCode that is selected on screen start?

Reply:
looks like no, but i'm not sure. 
I just reverted back changes.

Testing is okay, so looks like it's ok